### PR TITLE
sql: support SET datestyle TO ISO

### PR DESCRIPTION
**Pull request by asitm9**

### CODE REVIEW

File: pkg/sql/vars.go [hunk 1]
```diff
@@ -285,6 +285,24 @@ var varGen = map[string]sessionVar{
...
+		Get: func(*planner) string {
+			return "ISO"
+		},
+		Set: func(_ context.Context, p *planner, values []parser.TypedExpr) error {
+			s, err := p.getStringVal(`datestyle`, values)
+			if err != nil {
+				return err
+			}
+			upper := strings.ToUpper(s)
+			if upper != "ISO" {
```
Comment (justinj) [inline pkg/sql/vars.go]: Since there's only one option, might as well just inline `upper`:
```
if strings.ToUpper(s) != ...
```

**Review by justinj — COMMENTED** [review]
This looks pretty good to me! Could you also add a couple tests in `pkg/sql/logictest/testdata/logic_test/set`? Something like [this](https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/logictest/testdata/logic_test/set#L143-L147).

**Comment by asitm9** [comment]
1)I have added some more test cases  as suggested .
2) Code modification is done as per suggestion.

**Comment by asitm9** [comment]
Unable to edit the label to do-not-merge.

File: .gitignore [hunk 2]
```diff
@@ -19,6 +19,8 @@ artifacts
 # make stress, acceptance produce stress.test, acceptance.test
 *.test*
 
+.idea/
```
Comment (justinj) [inline .gitignore]: I think we don't want editor-specific stuff in the `.gitignore`, if you look at the top of this file there's instructions for how to add it to your global config.

**Review by justinj — COMMENTED** [review]
Besides the addition to the gitignore this looks good to me, @jordanlewis do you want to take a look?

File: .gitignore (hunk 2 above)
Comment (asitm9) [inline .gitignore]: `.gitignore` is reverted.

**Review by jordanlewis — APPROVED** [review]

**Comment by justinj** [comment]
Thanks for the contribution @asitm9!

**Comment by asitm9** [comment]
Always welcome @justinj . Looking forward to add more values to the product.

**Pull request merged**